### ISSUE TITLE
no public keys found for key provided

### ISSUE THREAD

**Issue by Bjorn-Eric-Abr** [picosh/pico#3]
Hi!

When trying to access (publish, admin) my blog using `ssh -i ~/.ssh/mykey prose.sh`
i get 
```
no public keys found for key provided
scp: Connection closed
```

Any suggestion on how to get in? Thanks
`

**Comment by Bjorn-Eric-Abr** [comment]
Hi,  still not getting in to my account / blog. Using a Makefile like @neurosnap does here https://github.com/neurosnap/prose-blog/blob/main/makefile so I am certain I am using the same SSH key.
Any ideas? Thanks in advance

**Comment by neurosnap** [comment]
Unfortunately if you are using an ssh-agent the `-i` won't always be preferred.  Can you share output from `ssh -vvv -i ~/.ssh/mykey prose.sh`?  That will confirm which key is being sent to prose.

**Comment by Bjorn-Eric-Abr** [comment]
No worries! Got a bit sad since I am kind of hyped about setting it up!

I upgraded to Ventura, could this have something to do with it?
https://apple.stackexchange.com/questions/451281/unable-to-scp-since-upgrade-to-ventura

Can I send the log privately? THANKS!

**Comment by neurosnap** [comment]
Send logs to me [at] erock.io

**Comment by neurosnap** [comment]
Another way to test this:

```bash
ssh-add -D # removes all keys in agent
ssh-add ~/.ssh/mykey # add key used for prose.sh
```

**Comment by Bjorn-Eric-Abr** [comment]
> Another way to test this:
> 
> ```shell
> ssh-add -D # removes all keys in agent
> ssh-add ~/.ssh/mykey # add key used for prose.sh
> ```

Tried this with no luck.

Sent you an email with the log - thank you _so_ much for taking the time to investigate this - appreciate it

**Comment by Bjorn-Eric-Abr** [comment]
Ok so I found the issue - 

TLDR; added `PubkeyAcceptedKeyTypes +ssh-rsa` to my `~/.ssh/config` after upgrading to Ventura.

I have had recently upgraded my Mac to Ventura, and with that came another version of SSH. [Solution can be found here](https://jay.gooby.org/2022/11/01/re-enable-older-rsa-ssh-keys-after-upgrading-to-macos-ventura).

**Comment by antoniomika** [comment]
I think we can bump go/x/crypto to solve this, will try and ask for feedback!

**Comment by antoniomika** [comment]
Managed to make a bunch of updates here and now ssh-rsa should be usable without the above, try it out and let me know how it works!

**Comment by Bjorn-Eric-Abr** [comment]
@antoniomika Yes, work without `PubkeyAcceptedKeyTypes +ssh-rsa` now. Good stuff!
